README.md

```diff
@@ -101,21 +101,30 @@ To solve this error, perform the following steps:
    npm start
    ```
 
-You can return to the [Project set-up](#project-set-up) section above and continue with the steps. You can also check [this issue in the Webpack GitHub repository](https://github.com/webpack/webpack/issues/14532) for more information about this error.
+1. Open a terminal.
+2. Navigate into the `ipfs-docs` folder:
 
-## Contribute to documentation
+   ```bash
+   cd ipfs-docs
+   ```
 
-We would **love ❤️ your help** to improve existing items or make new ones even better! [We also have bounties available](https://github.com/ipfs/devgrants/projects/1)!
+3. Run the following command:
+
+   ```shell
+    export NODE_OPTIONS=--openssl-legacy-provider
+   ```
 
-### Issues
+4. Run `npm start`.
```

Comment:
Numbers in an order list need to start with "1".

Reply:
Ok nice, changed those.